circus-train-core/src/main/java/com/hotels/bdp/circustrain/core/conf/CircusTrainTransformOptionsConfig.java

```diff
@@ -15,26 +15,15 @@
  */
 package com.hotels.bdp.circustrain.core.conf;
 
-import static com.hotels.bdp.circustrain.core.conf.TableParametersConfig.TABLE_REPLICATION_TABLE_PARAMETERS;
-
-import java.util.Map;
+import static com.hotels.bdp.circustrain.api.conf.TransformOptionsConfig.TRANSFORM_OPTIONS_PROPERTY;
 
 import org.springframework.boot.context.properties.ConfigurationProperties;
 import org.springframework.context.annotation.Configuration;
 
-@Configuration
-@ConfigurationProperties(prefix = TABLE_REPLICATION_TABLE_PARAMETERS)
-public class TableParametersConfig {
-
-  public static final String TABLE_REPLICATION_TABLE_PARAMETERS = "table-properties";
-
-  private Map<String, String> parameters;
+import com.hotels.bdp.circustrain.api.conf.TransformOptionsConfig;
 
-  public Map<String, String> getParameters() {
-    return parameters;
-  }
+@Configuration("transformOptionsConfig")
```

Comment:
Shouldn't this just be "transformOptions"?

Reply:
This value maps the configuration to a bean, so it should really match the bean the config is being mapped to. Now that the class has changed name, it should indeed be just `transformOptions`.